### ISSUE TITLE
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, in Publish screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#862]
**Describe the bug**
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, in Publish screen.

**To Reproduce**

1. Go to Publish  (http://localhost:9992/cm/app/?view=publish&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. Click on a Site.
2. Mouse hover on the Add New Server button. It will show "Add New Server" label.
3. Now try move the pointer over the "Add New Server, the label disappears.
4. User should be able to move pointer over the additional content (tool tip, label etc.).
 

**Expected behavior**
User should be able to move pointer over the additional content (tool tip, label etc.), and this additional content should not disappear, while moving focus on it.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
This is not a bug as there is no additional content on Hover. In application on Hover, only title is displayed. So closing this.
